app/views/shared/authorization_requests/show.html.erb

```diff
@@ -18,33 +18,184 @@
           <b><%= t('.delivered_at', humanized_date: friendly_format_from_timestamp(@authorization_request.created_at)) %></b>
           - 
           <%= link_to t('.links.to_datapass', external_id: @authorization_request.external_id).html_safe, 
-            datapass_authorization_request_url(@authorization_request), 
-            id: :authorization_request_link,
-            class: %w(fr-link fr-text--xs), 
-            target: '_blank' 
-          %>
+          datapass_authorization_request_url(@authorization_request), 
+          id: :authorization_request_link,
+          class: %w(fr-link fr-text--xs), 
+          target: '_blank' 
+        %>
         </span>
       </div>
     </div>
 
     <div id="habilitation_main_token_infos" class="fr-card__content">
-      <%=
-        render partial: 'shared/tokens/detail', 
+      <div class="fr-card__title">
+        <h2 class="fr-h3 fr-mb-n0-5v">
+          <%= t('.main_token.title') %>
+        </h2>
+      </div>
+      <div class="fr-card__desc">
+        <%= render partial: 'shared/tokens/detail', 
           locals: {
-            title: t('.main_token.title'),
             token: @main_token,
           }
-      %>
-
-      <% @banned_tokens.each do |banned_token| %>
-        <%=
-          render partial: 'shared/tokens/detail', 
+        %>
+        <% @banned_tokens.each do |banned_token| %>
+          <%= render partial: 'shared/tokens/detail', 
             locals: {
-              title: nil,
               token: banned_token,
             }
-        %>
-      <% end %>
+          %>
+        <% end %>
+        <ul class="fr-btns-group--right fr-btns-group--inline-sm fr-btns-group--inline-reverse fr-btns-group--icon-right">
+          <% if @authorization_request.demandeur == @current_user or @authorization_request.contact_technique == @current_user %>
```

Comment:
Vu que tu fais pas mal de logique dans la vue, je pense qu'il faudrait refactoriser ça.

Je vois 2 choix possibles ici:
1. Une facade -> `AuthorizationRequestFacade` avec des méthodes outils du genre `can_show_technical_user_modal?` and stuff
2. Une gestion de droits via un système comme [Pundit](https://github.com/varvet/pundit)

En l'état de ton code je partirai avec 1., cependant je pense que l'ensemble de tes modals plus bas devrait être refactorisé dans des actions spécifiques, et je pencherai donc plus pour 2.

Pour ce qui est du refactor des modals, il y a une mécanique existante dans le code qui exploite turbo pour afficher une modale (ou ouvrir une nouvelle page). 

Un exemple: https://entreprise.api.gouv.fr/catalogue/inpi/rne/beneficiaires_effectifs
Si tu cliques sur `[Aperçu d'un exemple de réponse JSON](https://entreprise.api.gouv.fr/catalogue/inpi/rne/beneficiaires_effectifs/exemple)` depuis cette page cela ouvre une modal, mais si tu l'ouvres dans un nouvel onglet c'est une autre page.

Je t'invite à refactoriser tout ton bloc de N modals avec cette mécanique, et limiter les actions via une gestion de droit en fonction des conditions que tu appliques.

Cette approche a plusieurs avantages:
1. C'est plus kiss et simple à tester
2. C'est rétrocompatible si tu n'as pas de js
3. On peut faire des liens directs vers les éléments